Get ready blog post for Portland 2021 (#1483)
writethedocs/www: docs/conf/portland/2021/news/events-and-activities.rst

:template: {{year}}/generic.html

> .. post:: Apr 12, 2021

Reviewer: ```suggestion
.. post:: Apr 15, 2021
```

   :tags: {{year}}, portland-{{year}}, tickets

Get Ready for the Conference!
=============================

Hey everyone!

We're just a couple of weeks away from the start of the conference, and we're getting excited to see you soon at our online venue for 3 days of content, collaboration, and conversation. We're happy to share with you some useful resources to help you prepare for the conference.

Ticket sales closing on April 22nd 2021
---------------------------------------

> To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **April 22nd, 2021**.

Reviewer: We should probably note a time here. Maybe 5pm Pacific?

> To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **April 22nd, 2021**.

Author: ```suggestion
To help us manage capacity and make sure that our conference environment is cozy and friendly, we are going to close ticket sales on **April 22nd, 2021 at 5PM Pacific Time**.
```

If you haven't purchased your ticket yet we invite you to `get your tickets soon <https://www.writethedocs.org/conf/portland/2020/tickets/>`_.

> For those of you who already purchased your tickets, you will get an email through Ti.to with the Hopin registration link and instructions on April 19th, 2021. 

Reviewer: We should mention that we’ll send another email and the date of that email too. 

Like “we will send an email with the login instructions on April 19 and again to all registered users on April 23”

> For those of you who already purchased your tickets, you will get an email through Ti.to with the Hopin registration link and instructions on April 19th, 2021. 

Author: ```suggestion
For those of you who already purchased your tickets, you will get an email through Ti.to with the Hopin registration link and instructions on April 19th, 2021. We will also send a reminder on April 23rd, in case you missed the first email.
```

Each year we create a guide to help our attendees get the most out of the conference.
Following the success of last year's conferences and based on your feedback, we expanded `our Welcome Wagon guide <https://www.writethedocs.org/conf/portland/2020/welcome-wagon/>`_ to help you get the most out of our event this year.

We have a lot of information about how to use Hopin, and lots of tips and tricks for the best way to plan your days.

This year we are also happy to announce that the Welcome Wagon team will hold a **live orientation session** for the virtual platform! The session will be held on Sunday during Writing Day, and will give you a chance to walk through the virtual event and ask all your questions. 

Check out the `Schedule page <https://www.writethedocs.org/conf/portland/{{year}}/schedule/>`_ for exact time and location.

All our events and activities
-----------------------------

Write the Docs is more than a conference. Every year we make sure to have a variety of activities that our attendees can enjoy, whether you want to learn something, meet new people, or share knowledge. And even though this years conference is held online, we're holding to that plan.

> * **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.

Reviewer: Should mention people sign up on the day of

> * **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required.

Author: ```suggestion
* **Lightning Talks.** We'll have `lightning talks <https://www.writethedocs.org/conf/portland/{{year}}/lightning-talks/>`__ each day after lunch. These are 5 minute presentations and a lot of fun. They can be about anything you want, this is your chance to share something you are passionate about with the audience. Slides are not required. Sign-up for lightning talks will be posted on Monday and Tuesday morning, and the selected talks will be announced before the lunch break on each day.. 
```

* **Unconference.** We'll be running the `Unconference <https://www.writethedocs.org/conf/portland/{{year}}/unconference/>`_ all day Monday and on Tuesday afternoon. Want to chat with like-minded folks about a specific thing? This is the place.
* **Job Fair**. As with the previous few years, on Tuesday morning we'll have a `job fair <https://www.writethedocs.org/conf/portland/{{year}}/job-fair>`_ where you can find your next documentarian friendly job.

> * **Social event**. On Monday night the conference platform will become the social event space. The main stage will be closed, the main focus will be attendee-created sessions. You're welcome to create sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.

Reviewer: Just needs to be updated to note that were using a different platform, not our existing platform. I think the previous email had some verbiage.

> * **Social event**. On Monday night the conference platform will become the social event space. The main stage will be closed, the main focus will be attendee-created sessions. You're welcome to create sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.

Author: ```suggestion
* **Social event**. On Monday night the main stage will be closed, and we will switch to social mode! We will be using a different platform for some of the social event, and we will post the link for the social space during the day on Monday. You're welcome to run sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.
```

> * **Social event**. On Monday night the main stage will be closed, and we will switch to social mode! We will be using a different platform for some of the social event, and we will post the link for the social space during the day on Monday. You're welcome to run sessions for board games, trivia quizzes, karaoke, or any other social activity that you want to share with your fellow documentarians.

Reviewer: ```suggestion
* **Social event**. On Monday night the main stage will be closed, and we will switch to social mode! We will be using a spatial chat platform for the social event this year, and we will post the link for the social space during the day on Monday. It will be much closer to a "real life" social where attendees can move between different rooms and group conversations, and the `Pac-Man Rule <https://www.ericholscher.com/blog/2017/aug/2/pacman-rule-conferences/>`_ even works!
```
